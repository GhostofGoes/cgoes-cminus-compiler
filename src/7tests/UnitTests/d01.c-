main()
{
    outputb(1<2);
    outputb(1>2);
    outputb(1==2);
    outputb(1!=2);
    outputb(1>=2);
    outputb(1<=2);
    outputb(2==2);
    outputb(2!=2);
    outputb(2>=2);
    outputb(2<=2);

    outputb(true==true);
    outputb(true!=true);
    outputb(false==true);
    outputb(false!=true);
    outputb(false==false);
    outputb(false!=false);
}
