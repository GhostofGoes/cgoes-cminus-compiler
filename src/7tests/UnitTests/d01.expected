Number of warnings: 0
Number of errors: 0
------------------------------------

Memory Configuration: Data Addresses: 0-9999  Instruction Addresses: 0-9999
Loading file: d01.tm
T F F T F T T F T T T F F T T F Status: Halted
Bye.
